tests/Http/Controllers/Api/Entries/Upload/Internal/UploadWebControllerBulkUploadsTest.php

```
@@ -0,0 +1,441 @@
+            //dd($e->getMessage(), $response, json_encode($entry), json_encode($projectDefinition));
+            $this->logTestError($e, $response);
+        }
+    }
+
+    public function test_should_not_touch_created_at_on_bulk_upload_loop()
+    {
+        $entriesCount = rand(2, 10);
+        for ($i = 0; $i < $entriesCount; $i++) {
+            $this->test_should_not_touch_created_at_on_bulk_upload();
+        }
+    }
```

PHPMD (2.15.0)

The method test_should_not_touch_created_at_on_bulk_upload_loop is not named in camelCase. (Controversial Rules)

(CamelCaseMethodName)